### PR TITLE
Admin user order#114

### LINKED ISSUE #114: User Story 23, Admin Sees User's Orders
```
As an admin user
When I visit a user's profile page
I see the same order data that the user sees
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #121

depends on #116

depends on #102

### PR DESCRIPTION
**Pull request by stoic-plus**
closes #114 

* Adds more testing but otherwise card was already done

### CODE REVIEW

File: spec/features/users/default/user_sees_profile_page_spec.rb [hunk 1]
```diff
@@ -54,9 +54,9 @@
         expect(page).to have_content("View Order ##{@o_1.id}")
         expect(page).to have_content("Created at: #{@o_1.created_at}")
         expect(page).to have_content("Updated at: #{@o_1.updated_at}")
-        expect(page).to have_content("#{@o_1.status}")
-        expect(page).to have_content("#{@o_1.quantity_of_order}")
-        expect(page).to have_content("#{@o_1.grand_total}")
+        expect(page).to have_content("Status: #{@o_1.status}")
+        expect(page).to have_content("Item Count: #{@o_1.quantity_of_order}")
+        expect(page).to have_content("Grand Total: #{@o_1.grand_total}")
```
Comment (le3ah) [inline spec/features/users/default/user_sees_profile_page_spec.rb]: Good call
